graphql-dgs-spring-webmvc/src/main/kotlin/com/netflix/graphql/dgs/mvc/DgsRestController.kt

```diff
@@ -247,19 +244,6 @@ open class DgsRestController(
             responseHeaders
         } else HttpHeaders()
 
-        val result = try {
-            TimeTracer.logTime(
-                { mapper.writeValueAsBytes(executionResult.toSpecification()) },
-                logger,
-                "Serialized JSON result in {}ms"
-            )
-        } catch (ex: InvalidDefinitionException) {
-            val errorMessage = "Error serializing response: ${ex.message}"
-            val errorResponse = ExecutionResultImpl(GraphqlErrorBuilder.newError().message(errorMessage).build())
-            logger.error(errorMessage, ex)
-            mapper.writeValueAsBytes(errorResponse.toSpecification())
-        }
-
-        return ResponseEntity(result, responseHeaders, HttpStatus.OK)
+        return ResponseEntity.ok().headers(responseHeaders).body(executionResult.toSpecification())
```

Comment:
@srinivasankavitha what do you think about this change?
> Instead of directly serializing the response to a String or bytes, return the response as a Map as built by
> ExecutionResult.toSpecification. Doing this allows the Jackson HttpMessageConverter to serialize the result directly to the response's OutputStream.

Reply:
Sorry for the delayed review. Looks good to me!